glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/minishift/minishift
 import:
 - package: github.com/docker/machine
-  version: 0.9.0
+  version: 0.11.0
```

Comment:
@hferentschik @LalatenduMohanty I think issue was there we had kubernetes dependencies but now it was quite simple only had to take care of systemctl unit file bits.